install/helm/open-match/values-production.yaml

```diff
@@ -190,6 +190,8 @@ open-match-core:
   queryPageSize: 10000
 
   redis:
+    # Duration for redis locks to expire.
```

Comment:
I think mutextTimeout should follow enabled parameter

Reply:
and when disabled, how should it be handled in the code??

Reply:
Yes, you are right. I suppose if section is not enabled that means all parameters defined in it are not used at all? Maybe it is better to move that parameter to open-match-core? and name parameter like backfillLockTimeout?

Reply:
Maybe put this configuration before enabled is kind of misleading. On the other hand, i like your suggestion, will change it